### PR TITLE
network: fix svace issues

### PR DESCRIPTION
**Pull request by pillip8282**
fix svace issues in network program.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ce1ab9a6ef7763fb40b5a2d0ec07676ea5b267b2] - Code Rule Check (C++) OK.

**Comment by lokeshbv** [comment]
@pillip8282 It will be helpful to understand , if you mention svace issues in commit message.

**Comment by pillip8282** [comment]
@lokeshbv 
Ok I'll add those informations

File: apps/system/utils/netcmd.c [hunk 1]
```diff
@@ -113,19 +118,33 @@ static void nic_display_state(void)
...
+				ndbg("fail %s:%d\n", __FUNCTION__, __LINE__);
+				continue;
+			}
 			printf("RUNNING: %s\n", (ifr->ifr_flags & IFF_UP) ? "UP" : "DOWN");
 		}
 		printf("\tinet addr: %s\t", inet_ntoa(sin->sin_addr));
 
-		ioctl(fd, SIOCGIFNETMASK, (unsigned long)ifr);
+		ret = ioctl(fd, SIOCGIFNETMASK, (unsigned long)ifr);
+		if (ret < 0) {
```
Comment (davidfather) [inline apps/system/utils/netcmd.c]: I think when ioctl fails, this util should not print mask set result.

File: apps/system/utils/netcmd.c [hunk 2]
```diff
@@ -113,19 +118,33 @@ static void nic_display_state(void)
...
+			ndbg("fail %s:%d\n", __FUNCTION__, __LINE__);
+		}
 		sin = (struct sockaddr_in *)&ifr->ifr_addr;
 		printf("Mask: %s\t", inet_ntoa(sin->sin_addr));
-		ioctl(fd, SIOCGIFMTU, (unsigned long)ifr);
+		ret = ioctl(fd, SIOCGIFMTU, (unsigned long)ifr);
+		if (ret < 0) {
```
Comment (davidfather) [inline apps/system/utils/netcmd.c]: I think when ioctl fails, this util should not print MTU set result.

**Comment by seinfra** [comment]
Target : [80d78813084c40d59931e1a7c4eb751977c24ca3] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [80d78813084c40d59931e1a7c4eb751977c24ca3] - Code Rule Check OK.

File: apps/system/utils/netcmd.c (hunk 1 above)
Comment (pillip8282) [reply apps/system/utils/netcmd.c]: thanks, I'll fix

File: apps/system/utils/netcmd.c (hunk 2 above)
Comment (pillip8282) [reply apps/system/utils/netcmd.c]: thanks, I'll fix it

**Comment by seinfra** [comment]
Target : [5eabd2ad5b7fbdb4a0bb3f82a2012500fa30b5c0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [5eabd2ad5b7fbdb4a0bb3f82a2012500fa30b5c0] - Code Rule Check (C++) OK.

**Pull request merged**